python/cuml/test/test_random_forest.py

```diff
@@ -560,12 +544,11 @@ def test_rf_regression_sparse(datatype, split_algo, mode, column_info,
         X, y = make_regression(n_samples=500, n_features=ncols,
                                n_informative=n_info,
                                random_state=123)
-
     elif mode == 'quality':
         X, y = fetch_california_housing(return_X_y=True)
 
     else:
-        X, y = make_regression(n_samples=100000, n_features=ncols,
+        X, y = make_regression(n_samples=3000, n_features=ncols,
```

Comment:
we can still reduce the n_samples value here, the point of this test is to see if the fil_sparse_format parameters are creating the right format of the fil tree given different `algo` values.